app/controllers/users/registrations_controller.rb

```diff
@@ -4,7 +4,13 @@
 class Users::RegistrationsController < Devise::RegistrationsController
   # POST /resource
   def create
+    if User.where(:email => params[:user][:email]).any?
```

Comment:
Can you change the where clause to use the hash syntax?  Just to say `email: params[:user][:email]`